source/main/index.ts

```diff
@@ -133,8 +135,50 @@ const handleWindowClose = async (event: Event | null | undefined) => {
   await safeExit();
 };
 
+const clearLedger = async () => {
+  const flagFileShortName = 'has_upgraded_to_pv_10';
+  const chainLedgerDir = 'chain/ledger';
+
+  const flagFileLongName = path.join(
+    launcherConfig.stateDir,
+    flagFileShortName
+  );
+
+  // Check if the flag file exists
+  if (fs.existsSync(flagFileLongName)) {
+    console.log(`${flagFileLongName} found. NoHskpg.`);
```

Comment:
Please use logger instead of console for all the places.
```suggestion
    logger.log(`${flagFileLongName} found. NoHskpg.`);
```